README.md

```diff
@@ -71,6 +71,7 @@ by the client itself.
 
 ### Events
 
+- client-initialized: Triggers when the drop-zone client have been initialized
```

Comment:
```suggestion
- client-initialized: Is triggered when the drop-zone client has been initialized. Use this event to customize the client.
```

Your part about the customization is fine 😄 I'd not get rid of that. The `drop-zone client` is third person, so `has`

Reply:
`Triggers` => local consistency 😂